.github/workflows/deploy-export-template.yaml

```diff
@@ -358,7 +358,6 @@ jobs:
           repository: godotengine/godot
           ref: 3.5.1-stable
 
-      # has to be the first step as the download url is only valid for 1 minute! See: https://stackoverflow.com/a/65049722
```

Comment:
Isn't this the case anymore then?

Reply:
IIRC that's the case only between workflows